Userland/Libraries/LibJS/Runtime/Temporal/AbstractOperations.cpp

```diff
@@ -1079,60 +1079,83 @@ ThrowCompletionOr<ISODateTime> parse_iso_date_time(GlobalObject& global_object,
 
     // 1. Assert: Type(isoString) is String.
 
-    // 2. Let year, month, day, hour, minute, second, fraction, and calendar be the parts of isoString produced respectively by the DateYear, DateMonth, DateDay, TimeHour, TimeMinute, TimeSecond, TimeFraction, and CalendarName productions, or undefined if not present.
+    // 2. Let year, month, day, fraction, and calendar be the parts of isoString produced respectively by the DateYear, DateMonth, DateDay, TimeFraction, and CalendarName productions, or undefined if not present.
     auto year_part = parse_result.date_year;
     auto month_part = parse_result.date_month;
     auto day_part = parse_result.date_day;
+    auto fraction_part = parse_result.time_fraction;
+    auto calendar_part = parse_result.calendar_name;
+
+    // 3. Let year be the part of isoString produced by the DateYear production.
+    // NOTE: Duplicate assignment, already covered above (spec issue)
```

Comment:
I assume you'll open an issue? :)

Reply:
https://github.com/tc39/proposal-temporal/pull/1987